mathemagic.py

```diff
@@ -29,9 +36,12 @@
             random_num = ('20' + str(random_number))
         else:
             random_num = ('2' + str(random_number)) 
+#The sum of all randoms numbers is predicted at the start of the first input            
```

Comment:
This comment should be indented. Also, there is a typo in there. The comment should read: 

> The sum of all random numbers is predicted at the start of the first input